shared-libs/cht-datasource/test/local/person.spec.ts

```diff
@@ -87,100 +87,76 @@ describe('local person', () => {
       });
 
       it('returns null if the doc returned is not a person', async () => {
-        const person = { type: 'person', _id: 'uuid', _rev: 'rev' };
-        const place0 = { _id: 'place0', _rev: 'rev' };
-        const place1 = { _id: 'place1', _rev: 'rev' };
-        const place2 = { _id: 'place2', _rev: 'rev' };
-        getLineageDocsByIdInner.resolves([person, place0, place1, place2]);
+        const notPerson = { type: 'not-person', _id: 'uuid', _rev: 'rev' };
+        const mockFunction = sinon.stub().resolves(notPerson);
+        mockFetchHydratedDoc.returns(mockFunction);
         isPerson.returns(false);
         settingsGetAll.returns(settings);
 
         const result = await Person.v1.getWithLineage(localContext)(identifier);
 
         expect(result).to.be.null;
-        expect(getLineageDocsByIdInner.calledOnceWithExactly(identifier.uuid)).to.be.true;
-        expect(isPerson.calledOnceWithExactly(settings, person)).to.be.true;
+        expect(mockFetchHydratedDoc.calledOnceWithExactly(localContext.medicDb)).to.be.true;
+        expect(mockFunction.calledOnceWithExactly(identifier.uuid)).to.be.true;
+        expect(isPerson.calledOnceWithExactly(settings, notPerson)).to.be.true;
         expect(warn.calledOnceWithExactly(`Document [${identifier.uuid}] is not a valid person.`)).to.be.true;
         expect(debug.notCalled).to.be.true;
-        expect(getContactLineageInner.notCalled).to.be.true;
-        expect(getContactLineageOuter.calledOnceWithExactly(localContext.medicDb)).to.be.true;
-        expect(getLineageDocsByIdOuter.calledOnceWithExactly(localContext.medicDb)).to.be.true;
       });
 
       it('returns a person if no lineage is found', async () => {
         const person = { type: 'person', _id: 'uuid', _rev: 'rev' };
-        getLineageDocsByIdInner.resolves([person]);
+        const mockFunction = sinon.stub().resolves(person);
+        mockFetchHydratedDoc.returns(mockFunction);
         isPerson.returns(true);
         settingsGetAll.returns(settings);
 
         const result = await Person.v1.getWithLineage(localContext)(identifier);
 
         expect(result).to.equal(person);
-        expect(getLineageDocsByIdInner.calledOnceWithExactly(identifier.uuid)).to.be.true;
+        expect(result).to.deep.equal(person);
+        expect(mockFetchHydratedDoc.calledOnceWithExactly(localContext.medicDb)).to.be.true;
+        expect(mockFunction.calledOnceWithExactly(identifier.uuid)).to.be.true;
         expect(isPerson.calledOnceWithExactly(settings, person)).to.be.true;
         expect(warn.notCalled).to.be.true;
-        expect(debug.calledOnceWithExactly(`No lineage places found for person [${identifier.uuid}].`)).to.be.true;
-        expect(getContactLineageInner.notCalled).to.be.true;
-        expect(getContactLineageOuter.calledOnceWithExactly(localContext.medicDb)).to.be.true;
-        expect(getLineageDocsByIdOuter.calledOnceWithExactly(localContext.medicDb)).to.be.true;
       });
```

Comment:
```suggestion
```

suggestion: can just remove this whole test case since the logic does not exist in cht-datasource anymore.